pytorch_lightning/accelerators/accelerator_connector.py

```diff
@@ -186,6 +187,9 @@ def select_accelerator(self):
         if self.accelerator is not None and isinstance(self.accelerator, Accelerator):
             self.accelerator.trainer = self.trainer
             self.accelerator.ddp_plugin = self.trainer.plugin_connector.ddp_plugin
+            if type(self.accelerator) == PipeAccelerator:
```

Comment:
Hacky as hell for now...

Reply:
Maybe worth to refactor/cleanup this part. It would look smoother.